src/main/java/view/FridgeView.java

```
@@ -67,7 +65,28 @@ public FridgeView(FridgeViewModel viewModel) {
             inputField.setText("");
         });
 
-        // 监听 Back
+        deleteButton.addActionListener(e -> {
+            int idx = ingredientList.getSelectedIndex();
+            if (idx < 0) {
+                JOptionPane.showMessageDialog(this,
+                        "Please select an ingredient to delete.",
+                        "No Selection",
+                        JOptionPane.WARNING_MESSAGE);
+                return;
+            }
+
+            Ingredient selectedIng = currentIngredients.get(idx);
+
+            try {
+                controller.deleteIngredient(selectedIng);
+            } catch (Exception ex) {
```

The current implementation uses a broad `catch (Exception ex)` block.  
To improve clarity and error handling, it would be better to catch the specific exceptions that `deleteIngredient` may throw.  
This makes the failure cases more explicit, prevents accidentally masking unexpected errors, and aligns with best practices for exception handling.